modules/customize-certificates-api-add-default.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/api-server.adoc
+
+[id="add-default-api-server-{context}"]
+= Add an API server default certificate
+
+The default API server certificate is issued by an internal {product-title} 
+cluster CA. This certificate can be replaced so that any API clients, such
+as `oc`, will use the defined certificate.
+
+.Prerequisites
+
+* You must have a valid certificate and key. 
+
+.Procedure
+
+. Create a secret that contains the certificate and key in the 
+`openshift-config` namespace.
++
+----
+$ oc create secret tls <certificate> --cert=</path/to/cert.crt> --key=</path/to/cert.key> -n openshift-config
+----
+
+. Update the API server to reference the created secret.
++
+----
+$ oc patch apiserver cluster --type=merge -p "{\"spec\": {\"servingCerts\": {\"defaultServingCertificate\": {\"name\": \"<certificate>\"}}}}"
```

Comment:
What namespace are you doing this in?

Reply:
The `oc patch` command for the ingress controller specifies a namespace. The apiserver does not require one. I've tested this both without a namespace and with one, and received confirmation that one is not required.